src/components/ContentActions/index.js

```diff
@@ -72,6 +141,38 @@ function ContentActions({ apiUri, queryParams, type, ...props }) {
               </Button>
             ))}
           </Grid>
+
+          <Grid item className={classes.filterSection}>
+            <Sort
+              key={value}
+              label="Sort By:"
+              name="ordering"
+              handleSelection={onSelection}
+              menuItems={[
+                { name: "Created At", value: "created_at" },
+                { name: "Deleted At", value: "deleted_at" },
+                { name: "Owner Screen Name", value: "owner__screen_name" },
+              ]}
+              value={value}
+            >
+              <Button
+                color="default"
+                variant="text"
+                size="large"
+                onClick={toggleIsDesc}
+                classes={{
+                  root: classes.buttonIcon,
+                }}
+                startIcon={
+                  <Image
+                    layout="fill"
+                    src={isDesc ? SortDown : SortUp}
+                    className={classes.large}
+                  />
+                }
+              />
```

Comment:
1. Since it a button with icon only i.e. without text, use [`IconButton`](https://v4.mui.com/components/buttons/#icon-buttons)
2. It should be part of `Sort` component (unless there is a reason I'm not aware of).

Reply:
@kilemensi Okay, um, I think there is another component that will rely on the Sort component but I can add a prop to disable the button.

Reply:
If there is another component @nerdlet just tell me we're reusing this button here and here, etc. My assumption is ordering (desc & asc) is tied to the sort drop-down.

Reply:
Ahhhh, okay, okay. I meant this here....we don't need a button for it.
<img width="1108" alt="Screenshot 2022-01-18 at 09 31 40" src="https://user-images.githubusercontent.com/3693563/149885835-ffc4dda5-6e88-4c5e-a047-7867d6682517.png">

Reply:
Why should a `Sort` component be able to accommodate `Show` component features?

Reply:
@kilemensi because the functionality + UI are the same for both input fields. The only difference is the button functionality that allows you to sort in `desc` or `asc`  for the `Sort`

Reply:
When you come across a situation where components have similar functionality @nerdlet then for UI, the best approach is **composition**. In this case:

You can create a Drop-down component as a stand-alone component since it's common to both `Show` and `Sort`. As an example, lets call this `TweetsSelect`.

Then you should ask yourself if there is any other commonality between `Show` and `Sort`. In this case, no. Then:

1. Implement `Show` using `TweetsSelect` by passing in `Show` specific props.
2. Implement `Sort` using `TweetsSelect` by passing in `Sort` specific props _as well as_ adding any other necessary functionality such as desc/asc button to `Sort`.

You can choose to make `TweetSelect` and `Show` just one component because `Show` doesn't add anything and reuses `TweetSelect` as is.

You shouldn't however make `Sort` (the more specific/complex component) _general_ so that it's reusable in `Show` (the more general/basic component) i.e. compose **simple** components to make **complex** ones and not the other way around.